torchaudio/io/_stream_reader.py

```diff
@@ -425,13 +425,34 @@ def get_out_stream_info(self, i: int) -> torchaudio.io.StreamReaderOutputStream:
         """
         return _parse_oi(self._be.get_out_stream_info(i))
 
-    def seek(self, timestamp: float):
+    def seek(self, timestamp: float, mode: str = "key"):
```

Comment:
Actually, I think we should change the default mode to "precise". That's the behavior I intended in the original implementation and that's what most DL practitioners would expect in their use cases.

Then we can cherry-pick this into the release as bug fix/new feature.